static-assets/components/cstudio-forms/controls/file-name.js

```diff
@@ -342,10 +351,16 @@ YAHOO.extend(CStudioForms.Controls.FileName, CStudioForms.CStudioFormField, {
     controlWidgetContainerEl.appendChild(inputContainer);
 
     var inputEl = document.createElement('input');
+    inputEl.setAttribute('autocomplete', 'off');
     this.inputEl = inputEl;
     YAHOO.util.Dom.addClass(inputEl, 'datum');
     YAHOO.util.Dom.addClass(inputEl, 'cstudio-form-control-input');
     YAHOO.util.Dom.addClass(inputEl, 'cstudio-form-control-file-name');
+
+    inputEl.onkeyup = () => this.adjustInputsWidth(inputEl, pathEl);
+    inputEl.onchange = () => this.adjustInputsWidth(inputEl, pathEl);
+    pathEl.onclick = () => inputEl.focus();
```

Comment:
@jvega190 clicking the div that contains both elements should also focus on the input.

Reply:
@jvega190 👆🏻